apps/200-flexdashboard-render-text/index.Rmd

```diff
@@ -62,55 +50,98 @@ shinyjster::shinyjster_js("
     })
   }
 
-  assert_equal_qs('a[href=\"#section-p1\"]', 'Page 1: Feb');
-  assert_equal_qs('a[href=\"#section-p2\"]', 'Page 2: Feb');
-  assert_equal('sidebarContent', 'Sidebar Content: Feb');
-
-  assert_equal_qs('#section-hidden1 .value-output', 'VB1: Feb');
-  assert_equal_qs('#section-hidden2 .value-output', 'VB2: Feb');
-  assert_equal_qs('#section-hidden3 .value-output', 'VB3: Feb');
+  function assert_page(idx, name) {
+    jst.add(Jster.shiny.waitUntilStable);
 
-  assert_equal_qs('#section-row2 .chart-title', 'Row 2 Title: Feb');
-  assert_equal_qs('#section-row2 p', 'Row 2 Content: Feb');
-  assert_equal_qs('#section-row2 .chart-notes', 'Caption: Feb');
+    jst.add(function() {
+      // make sure on first page
+      $('a[href=\"#section-p1\"]').click();
+      // make sure first tab is displayed
+      $('a[href=\"#section-tab1\"]').click();
```

Comment:
As we discussed, is there a better way to check that we are on the tab we wanted to be (in this case first tab) without using the `click()` function? Currently, it gives the assumption that we are clicking on section-tab1.